Dockerfile

```diff
@@ -68,7 +68,8 @@ RUN pip install /tmp/bzt-src \
 RUN bzt /tmp/bzt-src/examples/all-executors.yml -o settings.artifacts-dir=/tmp/all-executors-artifacts -sequential || (cat /tmp/all-executors-artifacts/webdriver-1.log; exit 1)
 
 RUN mkdir /bzt-configs \
-  && rm -rf /tmp/*
+  && rm -rf /tmp/* \
+  && mkdir /tmp/artifacts
```

Comment:
I seemed to be getting errors that this directory did not exist if I didn't mount a volume for it when executing `docker run`. Perhaps the old CMD was able to generate it whereas the new ENTRYPOINT just expects it to be there?